app/code/core/Mage/Rss/data/rss_setup/data-install-1.6.0.0.php

```diff
@@ -25,7 +25,7 @@
  */
 
 
-/* @var $installer Mage_Core_Model_Resource_Setup */
+/* @var Mage_Core_Model_Resource_Setup $installer */
```

Comment:
Double star

Reply:
@tmotyl can this be fixed later? Did a quick search and found +1.1k occurrences ...

Reply:
Sure

Reply:
Approve? ;)